<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>config_dialog_base</class>
 <widget class="QDialog" name="config_dialog_base">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>608</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>xgConstraintChecker Configuration</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/plugins/xgConstraintChecker/checker_config.png</normaloff>:/plugins/xgConstraintChecker/checker_config.png</iconset>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QDialogButtonBox" name="button_box">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>550</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="grp_xgApps">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>141</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="title">
    <string>xgApps Folders</string>
   </property>
   <widget class="QLabel" name="lbl_xgApps">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>211</width>
      <height>16</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>Enter the path to the local xgApps folder:</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="txt_xgApps_local">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>361</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_xgApps_network">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>231</width>
      <height>16</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>Enter the path to the network xgApps folder:</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="txt_xgApps_network">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>361</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="grp_results_db">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>381</width>
     <height>381</height>
    </rect>
   </property>
   <property name="title">
    <string>Results Database</string>
   </property>
   <widget class="QComboBox" name="cbo_db_type">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>PostGIS</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Spatialite</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>SQL Server</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="lbl_db_type">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Database Type:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_db">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Database:</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="txt_db">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>110</y>
      <width>271</width>
      <height>21</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="txt_port">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>80</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="txt_host">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>50</y>
      <width>271</width>
      <height>21</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_port">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Port:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_host">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Host / Instance:</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="grpLogin">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>361</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>Login</string>
    </property>
    <widget class="QCheckBox" name="chk_trusted">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>121</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Trusted Connection</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_user">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>User:</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_pwd">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Password:</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="txt_user">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>40</y>
       <width>191</width>
       <height>21</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="plainText">
      <string/>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="txt_pwd">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>70</y>
       <width>191</width>
       <height>21</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="grp_results_table">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>361</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>Results Table</string>
    </property>
    <widget class="QLabel" name="lbl_table">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Table name:</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="txt_table">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>50</y>
       <width>191</width>
       <height>21</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="plainText">
      <string/>
     </property>
    </widget>
    <widget class="QRadioButton" name="rb_new">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Create new</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="rb_existing">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>20</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Use existing</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="txt_geom">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>90</y>
       <width>191</width>
       <height>21</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="plainText">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_geom">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Geometry column:</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>txt_xgApps_local</tabstop>
  <tabstop>cbo_db_type</tabstop>
  <tabstop>txt_host</tabstop>
  <tabstop>txt_port</tabstop>
  <tabstop>txt_db</tabstop>
  <tabstop>chk_trusted</tabstop>
  <tabstop>txt_user</tabstop>
  <tabstop>txt_pwd</tabstop>
  <tabstop>button_box</tabstop>
 </tabstops>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>button_box</sender>
   <signal>accepted()</signal>
   <receiver>config_dialog_base</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>50</x>
     <y>550</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_box</sender>
   <signal>rejected()</signal>
   <receiver>config_dialog_base</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>50</x>
     <y>550</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbo_db_type</sender>
   <signal>currentIndexChanged(QString)</signal>
   <receiver>config_dialog_base</receiver>
   <slot>enableDBFields()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>170</x>
     <y>130</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chk_trusted</sender>
   <signal>stateChanged(int)</signal>
   <receiver>config_dialog_base</receiver>
   <slot>enableLoginFields()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>56</x>
     <y>268</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>251</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rb_existing</sender>
   <signal>toggled(bool)</signal>
   <receiver>txt_table</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>184</x>
     <y>438</y>
    </hint>
    <hint type="destinationlabel">
     <x>184</x>
     <y>473</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rb_existing</sender>
   <signal>toggled(bool)</signal>
   <receiver>txt_geom</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>150</x>
     <y>439</y>
    </hint>
    <hint type="destinationlabel">
     <x>146</x>
     <y>515</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>enableDBFields()</slot>
  <slot>enableLoginFields()</slot>
 </slots>
</ui>
